python/sync_with_qdrant/src/utils.py

```diff
@@ -0,0 +1,74 @@
+import os
+from appwrite.client import Client
+from appwrite.services.databases import Databases
+
+__dirname = os.path.dirname(os.path.abspath(__file__))
+static_folder = os.path.join(__dirname, "../static")
+
+
+def throw_if_missing(obj: dict, keys: list[str]) -> None:
+    """
+    Throws an error if any of the keys are missing from the object
+
+    Parameters:
+        obj (dict): Dictionary to check
+        keys (list[str]): List of keys to check
+
+    Raises:
+        ValueError: If any keys are missing
+    """
+    missing = [key for key in keys if key not in obj or not obj[key]]
+    if missing:
+        raise ValueError(f"Missing required fields: {', '.join(missing)}")
+
+
+def get_static_file(file_name: str) -> str:
+    """
+    Returns the contents of a file in the static folder
+
+    Parameters:
+        file_name (str): Name of the file to read
+
+    Returns:
+        (str): Contents of static/{file_name}
+    """
+    file_path = os.path.join(static_folder, file_name)
+    with open(file_path, "r") as file:
+        return file.read()
+
+def get_all_documents():
```

Comment:
Can we put this function inside a `appwrite.py` instead? It doesn't feel related to these other functions

Reply:
[`25a6fc1` (#289)](https://github.com/appwrite/templates/pull/289/commits/25a6fc1a0f800ffcdef0d61784e3b94719239661)